common/file/UtilTest.cpp

```diff
@@ -90,3 +100,54 @@ void UtilTest::testFilenameFromPath() {
   OLA_ASSERT_EQ(string("baz"), FilenameFromPathOrPath("/foo/bar/baz"));
 }
 
+/*
+ * Inline helper function to check a string ending for a match
+ */
+inline static bool endsWith(const std::string& str, const std::string& suffix) {
+  return (str.size() >= suffix.size()) &&
+          (0 == str.compare(str.size()-suffix.size(), suffix.size(), suffix));
+}
```

Comment:
Any reason not to use StringEndsWith?:
https://github.com/OpenLightingProject/ola/blob/0.10/include/ola/StringUtils.h#L91-L97

Reply:
No reason, I was just not aware. Thanks for pointing it out to me, changed that :+1: